content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2023
+linkTitle: KubeCon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun
+  cSpell:ignore Alolita Sharma Dyla Benedikt Bongartz Oliveira Pathak
+  cSpell:ignore Jaglowski observ
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, members of the governance committee, and
+technical committee are excited to be at [KubeCon EU][] in Amsterdam from April
+18 - 21, 2023!
+
+Read on to learn about all the things related OpenTelemetry during KubeCon.
+
+## KubeCon Talks and Maintainer Sessions
+
+- **[Jaeger: The Future with OpenTelemetry and Metrics](https://sched.co/1HySf)**<br>
+  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
+  11:55 - 12:30
+- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
+  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
+- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
+  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
+  April 19 • 14:30 - 15:05
+- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
+  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
+- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
+  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
+  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
+- **[Connected Observability Pipelines in the OpenTelemetry Collector](https://sched.co/1HyXb)**<br>
+  by Daniel Jaglowski, observIQ<br> Friday, April 21 • 11:55 - 12:30
+- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
+  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
+  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
+  17:30
+
+## Co-located Events
+
+Come network with OpenTelemetry maintainers and core contributors during the
+[OpenTelemetry project meeting](https://sched.co/1JWS7), on Tuesday April 18,
+2023 from 13:00 - 15:00. You can attend with a _standard in-person pass_.
+
+[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
+of cloud-native observability projects_. This event will be held on April 18,
+2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
+
+> <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You
```

Comment:
Please add that observability day will also be live-streamed if you have a virtual ticket

Reply:
fixed